compiler/passes/normalize.cpp

```diff
@@ -779,7 +779,7 @@ static void insertRetMove(FnSymbol* fn, VarSymbol* retval, CallExpr* ret) {
                       new CallExpr(PRIM_COERCE, ret_expr,
                         fn->retExprType->body.tail->copy())));
   }
-  else if (fn->hasFlag(FLAG_IF_EXPR_FN))
+  else if (fn->hasFlag(FLAG_MAYBE_REF))
   {
```

Comment:
Yes it is. Without this change, test/classes/ferguson/forwarding/d-record-to-record-nocopy.chpl does not pass, because the ForwardingStmt::toFnDef will always return a value. In that test, we're returning a field accessor expression, which is naturally a ref. We want it to stay a ref if it starts out a ref.